app/assets/javascripts/common/my_facilities.js

```diff
@@ -48,6 +48,7 @@ function createBaseTrendChartConfig() {
       elements: {
         line: {
           borderJoinStyle: "round",
+          tension: 0.4,
```

Comment:
We're building up the tension I see... what does `tension` do?

Reply:
Tension controls how curvy a line is on a line graph when it crosses through a point.
0 means that there is no curve and 1 means its very curvy.
In v2.8 the default tension is 0.4, but in v3.9.1 the default tension has been changed to 0.
This is added to keep design consistency.
v3.9 default - [Line configuration defaults](https://www.chartjs.org/docs/3.9.1/configuration/elements.html#line-configuration)
You can 'increase' the tension here: https://www.chartjs.org/docs/3.9.1/charts/line.html - Click the 'setup' tab to find the tension option